src/client-functions/selectors/selector-attribute-filter.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
```

Comment:
Why it's named select-or-attribute? What it's even mean? Also, maybe it makes sense putting these in separate `filters` dir?